src/main/scala/com/spotify/spark/bigquery/BigQueryClient.scala

```diff
@@ -109,8 +110,29 @@ private[bigquery] class BigQueryClient(conf: Configuration) {
    */
   def load(gcsPath: String, destinationTable: TableReference,
            writeDisposition: WriteDisposition.Value = null,
-           createDisposition: CreateDisposition.Value = null): Unit = {
+           createDisposition: CreateDisposition.Value = null,
+           isPartitionedByDay: Boolean = false): Unit = {
```

Comment:
This client is not normally exposed to Spark users so it's OK to compromise the API a bit and keep it this way instead of adding value?
OTOH can we add a different method on `RDD`, i.e. `saveAsPartitionedBigQuery`?